cfme/utils/appliance/__init__.py

```diff
@@ -18,6 +18,7 @@
 import pytz
 import requests
 import sentaku
+from aenum import Enum
```

Comment:
Should get this from python enum, not aenum. 
`from enum import Enum`

Reply:
Done